src/endpoints/tokens/token.service.ts

```diff
@@ -280,7 +280,7 @@ export class TokenService {
 
       if (isCustomThumbnail === true) {
         nft.thumbnailUrl = `${this.apiConfigService.getMediaUrl()}/nfts/thumbnail/custom/${nft.identifier}`;
-      } if (isStandardThumbnail === true) {
+      } else if (isStandardThumbnail === true) {
```

Comment:
the custom thumbnail must have precedence, and in this situation, the custom thumbnail url will never be available, since the standard thumbnail will be taken with priority

Reply:
Adding the `else` keyword here assures that you say:
```custom > standard > defaults```

Without it, the URL containing "/custom" will never be used because it will be overridden in one of the following 3 lines starting with `nft.thumbnailUrl = ...` .